iOS/Flows/Home/_New/ConversationViewControllerDataSource+CellRegistration.swift

```diff
@@ -25,16 +25,18 @@ extension ConversationCollectionViewDataSource {
             guard let message = messageController.message else { return }
             let dataSource = item.dataSource
 
-            // Configure the cell
-            cell.contentView.set(backgroundColor: .red)
 
             if message.type == .deleted {
-                cell.textView.text = "DELETED"
-                cell.replyCountLabel.text = nil
+                cell.setIsDeleted()
             } else {
-                cell.textView.text = message.text
-                cell.replyCountLabel.setText("\(message.latestReplies.count)")
```

Comment:
Moving some of the cell setup code into the cell itself. Importantly, the cell still does not know anything about chat message objects or the data model in general.

Reply:
Lets be carefully to not store any state of the model on the cell itself. Im pretty sure it has a different lifecycle than the registration.